Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "built"
-version = "0.5.1"
+version = "0.5.2"
```

Comment:
Please don't bump the version in a PR, this is done separately.

Reply:
Sorry, I was using this as a git dependency for another project. Will change it back.